modules/admin_manual/pages/maintenance/manual_upgrade.adoc

```diff
@@ -32,7 +33,139 @@ With those steps completed, stop your webserver.
+
+We described how to backup your existing ownCloud installation at the beginning of this document. You will find it in `/var/www/owncloud-old-version-number`.
+
+Extract the unpacked ownCloud server directory and its contents to the location of your original ownCloud installation.
+
+[source,console]
+----
+tar -C /var/www/ -xvf owncloud-10.1.1.tar.bz2
+----
+
+With the new source files now in place of the old ones, next copy the `config.php` file from your old ownCloud directory to your new ownCloud directory:
+
+[source,console]
+----
+sudo cp /var/www/owncloud-old/config/config.php /var/www/owncloud/config/config.php
+----
+
+If you keep your `data/` directory _inside_ your `owncloud/` directory, move it from your old version of ownCloud to your new version:
+
+[source,console]
+----
+mv /var/www/owncloud-old-version-number/data /var/www/owncloud/data
+----
+
+If you keep it _outside_ of your `owncloud/` directory, then you don’t have to do anything with it, because its location is configured in your original `config.php`, and none of the upgrade steps touch it.
+
+=== Market and Marketplace App Upgrades
+
+Before getting too far into the upgrade process, please be aware of how the Market app and its configuration options affect the upgrade process.
+
+NOTE: The Market app — and other apps from the Marketplace — will not be updated when you upgrade ownCloud, if `upgrade.automatic-app-update` is set to `true` in config.php.
+
+In addition, if there are installed apps (whether compatible or incompatible with the next version, or missing source code) and the Market app is enabled but there is no available internet connection, these apps will need to be manually updated once the upgrade is finished.
+
+=== Copy Old Apps
+If you are using third party or enterprise applications, look in your new `/var/www/owncloud/apps/` directory to see if they are present. If not, copy them from your old `apps/` directory to your new one.
+
+=== Setting the Ownership for Upgrading
+To finalize the preparation of the upgrade, you need to set the correct ownership of the new ownCloud files and folders. Here is a neat little bash-script to do this on the fly. You will need to change the paths and webserver users according to your installation and/or Linux Distribution.
+
+Take your favourite editor and create the `oc_permit.sh` and copy the contents from beneath.
+
+[source,bash]
+----
+include::example$configuration/oc_permit.sh[]
+----
+
+Save and close the file and make it executable by running `chmod +x oc_permit.sh`.
+Then, run `sh oc_permit.sh`.
+You can find a more sophisticated script right here: xref:installation/manual_installation.adoc#set-strong-directory-permissions[strong permissions].
+
+=== Start the Upgrade
+
+With the apps disabled and ownCloud in maintenance mode, start xref:configuration/server/occ_command.adoc#command-line-upgrade[the upgrade process] from the command line:
+
+[source,console]
+----
+# Here is an example on Ubuntu Linux. 
```

Comment:
`#` is a first level header = title header. Is this intended?

Reply:
'#' within a console block it is a comment